jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -353,74 +353,68 @@ public Connection getConnection(String propertyKey, InterpreterContext interpret
-                StringBuilder connectionUrl = new StringBuilder(url);
-                appendProxyUserToURL(connectionUrl, user);
-                connection = getConnectionFromPool(connectionUrl.toString(),
-                        user, propertyKey, properties);
-              } else {
-                UserGroupInformation ugi = null;
-                try {
-                  ugi = UserGroupInformation.createProxyUser(
-                          user, UserGroupInformation.getCurrentUser());
-                } catch (Exception e) {
-                  logger.error("Error in getCurrentUser", e);
-                  StringBuilder stringBuilder = new StringBuilder();
-                  stringBuilder.append(e.getMessage()).append("\n");
-                  stringBuilder.append(e.getCause());
-                  throw new InterpreterException(stringBuilder.toString());
-                }
-
-                final String poolKey = propertyKey;
-                try {
-                  connection = ugi.doAs(new PrivilegedExceptionAction<Connection>() {
-                    @Override
-                    public Connection run() throws Exception {
-                      return getConnectionFromPool(url, user, poolKey, properties);
-                    }
-                  });
-                } catch (Exception e) {
-                  logger.error("Error in doAs", e);
-                  StringBuilder stringBuilder = new StringBuilder();
-                  stringBuilder.append(e.getMessage()).append("\n");
-                  stringBuilder.append(e.getCause());
-                  throw new InterpreterException(stringBuilder.toString());
-                }
+              UserGroupInformation ugi = null;
+              try {
+                ugi = UserGroupInformation.createProxyUser(
+                    user, UserGroupInformation.getCurrentUser());
+              } catch (Exception e) {
+                logger.error("Error in getCurrentUser", e);
+                StringBuilder stringBuilder = new StringBuilder();
+                stringBuilder.append(e.getMessage()).append("\n");
+                stringBuilder.append(e.getCause());
+                throw new InterpreterException(stringBuilder.toString());
+              }
+
+              final String poolKey = propertyKey;
+              try {
+                connection = ugi.doAs(new PrivilegedExceptionAction<Connection>() {
+                  @Override
+                  public Connection run() throws Exception {
+                    return getConnectionFromPool(connectionUrl, user, poolKey, properties);
+                  }
+                });
+              } catch (Exception e) {
+                logger.error("Error in doAs", e);
+                StringBuilder stringBuilder = new StringBuilder();
+                stringBuilder.append(e.getMessage()).append("\n");
+                stringBuilder.append(e.getCause());
+                throw new InterpreterException(stringBuilder.toString());
```

Comment:
why not just include e as inner exception like `InterpreterException(message, e)`?